app/components/collections/delete_button_component.rb

```diff
@@ -0,0 +1,29 @@
+# typed: false
+# frozen_string_literal: true
+
+module Collections
+  # Render the delete button for a collection
+  class DeleteButtonComponent < ApplicationComponent
+    CONFIRM_MESSAGE = 'Are you sure you want to delete this draft collection? It cannot be undone.'
```

Comment:
Is this button only available for draft collections? Or should this read `... delete this collection?`

Reply:
This is just a refactor, so I'm taking the existing text from https://github.com/sul-dlss/happy-heron/pull/1470/files#diff-251c3b4af41a707fdfad74ccb17786d67aad7db340ff96eb4a5c10310f87d975L3